crypto/packet/src/lib.rs

```diff
@@ -91,7 +91,7 @@ mod tests {
         );
 
         assert!(
-            hopr_packet_len < 1492 - 32,
+            hopr_packet_len < 1492 - 32, // 32 bytes was measured as the libp2p QUIC overhead
             "HOPR packet {hopr_packet_len} must fit within a layer 4 packet with libp2p overhead"
```

Comment:
Isn't it the layer 2 limitation?

Reply:
1492 is what you have on L4. 32 bytes is what you measured :P

Reply:
Yes, but the limitation stems from the L2 MTU, not the L4 itself, L4 can be larger.

Reply:
Sure, we do not assume L2 jumbo frames larger than 1500 bytes

Reply:
```suggestion
            "HOPR packet {hopr_packet_len} must fit within a layer 4 packet with libp2p overhead, considering a reasonable L2 MTU of 1492 bytes."
```

Reply:
Layer 2 is 1500, layer 4 is 1492